CHANGELOG.unreleased.md

```diff
@@ -33,6 +34,8 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- xen-api patch
```

Comment:
I don't see any change in the `xen-api`.

Reply:
Indeed, thx.